src/index.html

```diff
@@ -1,13 +1,319 @@
+    <!--#region HEAD  -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bang & Olufsen</title>
+    <link rel="icon" href="./icon/favicon.svg">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body" id="home">
+    <!-- #endregion -->
+    <!-- #region HEADER  -->
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#home">
+              <img src="img/logo.png" class="top-bar__logo"
+              alt="B&O logo">
+            </a>
+
+            <div class="top-bar__icons">
+              <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+
+          <div class="header__bottom">
+            <h1 class="header__title">
+              Your senses.
+              <br>
+              Set free.
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+    <!-- #endregion -->
+    <!-- #region MENUBAR -->
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top">
+            <div class="top-bar">
+              <a href="#home">
+                <img src="img/logo.png" class="top-bar__logo"
+                alt="B&O logo">
```

Comment:
if you have 3 or more attributes you should write them on the new line
```suggestion
                <img 
                  src="img/logo.png" 
                  class="top-bar__logo"
                  alt="B&O logo"
                >
```

Reply:
Lost this one( Done.